beaker-puppet.gemspec

```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |s|
   s.description = 'For use for the Beaker acceptance testing tool'
   s.license     = 'Apache-2.0'
 
-  s.required_ruby_version = '>= 2.7', '< 3.2'
+  s.required_ruby_version = '>= 2.7', '< 3.3'
```

Comment:
What are the issues with using Ruby >= 3.3? Is it just that we're not testing against those versions?

Reply:
ah yeah good point, I didn't think to add that. Updated.